res/values-vi/strings.xml

```diff
@@ -1040,8 +1092,10 @@
     <string name="Community">Cộng Đồng</string>
     <string name="Buy">Mua</string>
     <string name="Sell">Bán</string>
+    <string name="community_skin_information_for_seller" translatable="true">When you turn a custom skin into a community skin, it will appear at the &#x201c;Community Skins&#x201d; tab. Other people can see and buy your skin at the Community! Each time someone buys your skins in the community, you will receive a small plasma amount.</string>
     <string name="community_skin_requirement">Bạn cần phải mua tuỳ chỉnh skin để có thể mua skin cộng đồng.</string>
     <string name="You_can_purchase_this_community_skin_for_">Bạn có thể mua skin cộng đồng này với</string>
+    <string name="community_skin_information_for_buyer" translatable="true">Community skins are uploaded by other users. Community skins work just like custom skins and can be animated with default Nebulous skins.</string>
```

Comment:
Strings 1095 and 1098 are not based on the English file by the same reason mentioned.